main.asm

```diff
@@ -311,15 +311,27 @@ INCLUDE "data/battle_anims/subanimations.asm"
 INCLUDE "data/battle_anims/frame_blocks.asm"
 
 
+SECTION "BG Map Attributes (Debug)", ROMX
```

Comment:
I'm not sure if there's a smarter way to do this

Reply:
IMO this would be more clear, if it works:

Remove the `org $5000`:

```
ROMX $2f
	"Move Names"
	"BG Map Attributes"
```

Then in `main.asm`:

```
SECTION "BG Map Attributes", ROMX

IF DEF(_DEBUG)
    INCLUDE "engine/gfx/bg_map_attributes.asm"
    ds $5000 - @ ; org $5000
    INCLUDE "data/cgb/bg_map_attributes.asm"
ELSE
    ds $5000 - @ ; org $5000
    INCLUDE "data/cgb/bg_map_attributes.asm"
    INCLUDE "engine/gfx/bg_map_attributes.asm"
ENDC
```

Reply:
```
ERROR: main.asm(321):
    Expected constant expression: PC is not constant at assembly time
```
:p

Reply:
Drat, I tested that in <https://daid.github.io/rgbds-live/> but with a fixed `SECTION`. It's okay as-is then.